### PR TITLE
Add ci and cd

### PR DESCRIPTION
**Pull request by i1i1**
Signed-off-by: i1i1 <vanyarybin1@live.ru>

### CODE REVIEW

File: iroha2/sys/iroha_data_model/asset/__init__.py [hunk 1]
```diff
@@ -11,10 +11,10 @@
                                "iroha_data_model.asset.AssetDefinition"),
                               ("registered_by", "iroha_data_model.account.Id")]
 
-AssetValue = Enum[("Quantity", int), ("BigQuantity", int),
+AssetValue = Enum[("Quantity", int), ("BigQuantity", int), ("Fixed", float),
```
Comment (eadventurous) [inline iroha2/sys/iroha_data_model/asset/__init__.py:14]: Should `Fixed` be represented as float? I think it is better to use also a fixed point arithmetic lib in python.

File: generate/Cargo.lock [hunk 2]
```diff
@@ -1,1732 +0,0 @@
-# This file is automatically @generated by Cargo.
```
Comment (eadventurous) [inline generate/Cargo.lock:1]: Why did you decide to remove the Cargo.lock? Usually it is ok to remove lock for libs but to commit it for bins, and here it is a binary crate.

Comment (i1i1) [reply generate/Cargo.lock:1]: Because there is a single lock for a workspace

File: iroha2/sys/iroha_data_model/asset/__init__.py (hunk 1 above)
Comment (i1i1) [reply iroha2/sys/iroha_data_model/asset/__init__.py:14]: I dont think it matters that much for the first version of python library. We can change it later

File: generate/Cargo.lock (hunk 2 above)
Comment (eadventurous) [reply generate/Cargo.lock:1]: Oh, ok

File: iroha2/sys/iroha_data_model/asset/__init__.py (hunk 1 above)
Comment (eadventurous) [reply iroha2/sys/iroha_data_model/asset/__init__.py:14]: Ok, let's create an issue for that in python lib

Comment (eadventurous) [reply iroha2/sys/iroha_data_model/asset/__init__.py:14]: Also do you currently do any conversion between Fixed and float? I couldn't find the code, or is it for the later?

Comment (i1i1) [reply iroha2/sys/iroha_data_model/asset/__init__.py:14]: Hmm.. No i dont. I would do it later in pr with crypto primitives

Comment (eadventurous) [reply iroha2/sys/iroha_data_model/asset/__init__.py:14]: Ok, maybe create an issue for that to :)

**Review by eadventurous — APPROVED** [review]

**Pull request merged**